ui/test/cypress/helpers/create-spec.js

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env node
+const argv = require('yargs/yargs')(process.argv.slice(2))
```

Comment:
Can you add a comment here (or into Cypress README) explaining how to use this and when it's needed?

Reply:
Yes, done!